### ISSUE TITLE
Some problems with training

### ISSUE THREAD

**Issue by Shayebh** [amanchadha/iSeeBetter#8]
Hello, the idea of this paper is very inspiring to me, but there is a problem when reproducing your code, that is, the output SR image I trained are all black, the training code is unchanged, and the vimeO90K data set is also used. Have you ever encountered such a situation?

**Comment by amanchadha** [comment]
Hey,

I'd love to help you figure this out, but I'll need more detail to figure out what's up. Can you send me the parameters you're training with?

Also, note that in a GAN architecture, the generator starts generating noisy images while it works to get a grip over the data distribution (which the discriminator helps it to). This tells me that it seems that you haven't trained for a reasonable number of epochs -- how many did you train for?

Hope this helps.

**Comment by Shayebh** [comment]
Thank you very much for your recovery. I would like to ask you how many steps did you train when you were training?And do you fine-tune the weight of RBPN, or do you train from scratch? The parameters I used have not changed; they are all parameters in your code。

**Comment by wangyiwei0306** [comment]
@amanchadha Hi, I also encounter the same problem. I have trained about 25 epochs and then to test and find the results are colorful ｍosaic. The followings are my parameters:
`parser = argparse.ArgumentParser(description='Train iSeeBetter: Super Resolution Models')
parser.add_argument('--upscale_factor', type=int, default=4, help="super resolution upscale factor")
parser.add_argument('--batchSize', type=int, default=2, help='training batch size')
parser.add_argument('--testBatchSize', type=int, default=5, help='testing batch size')
parser.add_argument('--start_epoch', type=int, default=1, help='Starting epoch for continuing training')
parser.add_argument('--nEpochs', type=int, default=150, help='number of epochs to train for')
parser.add_argument('--snapshots', type=int, default=1, help='Snapshots')
parser.add_argument('--lr', type=float, default=1e-4, help='Learning Rate. Default=0.01')
parser.add_argument('--gpu_mode', type=bool, default=True)
parser.add_argument('--threads', type=int, default=8, help='number of threads for data loader to use')
parser.add_argument('--seed', type=int, default=123, help='random seed to use. Default=123')
parser.add_argument('--gpus', default=8, type=int, help='number of gpu')
parser.add_argument('--data_dir', type=str, default='./vimeo_septuplet/sequences')
parser.add_argument('--file_list', type=str, default='sep_trainlist.txt')
parser.add_argument('--other_dataset', type=bool, default=False, help="use other dataset than vimeo-90k")
parser.add_argument('--future_frame', type=bool, default=True, help="use future frame")
parser.add_argument('--nFrames', type=int, default=7)
parser.add_argument('--patch_size', type=int, default=64, help='0 to use original frame size')
parser.add_argument('--data_augmentation', type=bool, default=True)
parser.add_argument('--model_type', type=str, default='RBPN')
parser.add_argument('--residual', type=bool, default=False)
parser.add_argument('--pretrained_sr', default='RBPN_4x.pth', help='sr pretrained base model')
parser.add_argument('--pretrained', action='store_true', help='Use pretrained model')
parser.add_argument('--save_folder', default='weights/', help='Location to save checkpoint models')
parser.add_argument('--prefix', default='F7', help='Location to save checkpoint models')
parser.add_argument('--APITLoss', action='store_true', help='Use APIT Loss')
parser.add_argument('--useDataParallel', action='store_true', help='Use DataParallel')
parser.add_argument('-v', '--debug', default=False, action='store_true', help='Print debug spew.')`

And I also want to know those action='store_true' parameters are they true or false?

**Comment by amanchadha** [comment]
We fine-tuned RBPN for four epochs using both L2 loss and APIT loss.

Also, try the recommendations [here](https://github.com/amanchadha/iSeeBetter/issues/5#issuecomment-658545179).
